test_fms/time_interp/test_time_interp.F90

```diff
@@ -30,7 +30,7 @@ program test_time_interp
  type(time_type) :: Time_beg, Time_end, Time(num_Time)
  type(time_type), allocatable, dimension(:) :: Timelist
  integer :: index1, index2, mo, yr, timelist_len, outunit, ntest, nline
- real :: weight
+ real(TI_TEST_KIND_) :: weight
```

Comment:
It seems like the test is a guide on how to use time_interp.  Is there a way to check that the answers are as expected?

Reply:
I added checks for all but the last, since that one cycles through instead of going to set dates.